src/editors/VideoSelectorPage.jsx

```diff
@@ -2,25 +2,25 @@ import React from 'react';
 import { Provider } from 'react-redux';
 import PropTypes from 'prop-types';
 import ErrorBoundary from './sharedComponents/ErrorBoundary';
-import { VideoSelector } from './VideoSelector';
+import VideoSelector from './VideoSelector';
 import store from './data/store';
 
 const VideoSelectorPage = ({
   courseId,
   lmsEndpointUrl,
   studioEndpointUrl,
 }) => (
-  <ErrorBoundary>
-    <Provider store={store}>
+  <Provider store={store}>
+    <ErrorBoundary {...{ courseId, studioEndpointUrl }}>
       <VideoSelector
```

Comment:
Unrelated to your fix: this kind of thing breaks the abstraction of editors..... :( Why didn't they follow the pattern from the problem editor?